docs/source/Commands.rst

```diff
@@ -3,7 +3,7 @@
 Commands
 ========
 
-:doc:`Podman <markdown/podman.1>` (Pod Manager) Global Options
+:doc:`Podman <markdown/podman.1>` (Pod Manager) Global Options, Environment Variables, Exit Status, Files, and more
```

Comment:
Please change "Exit Status" to "Exit Status Codes" (or maybe just "Exit Codes"

Reply:
And maybe "Files" to "Configuration Files"?

Reply:
Exit Codes and Configuration Files it is